scripts/quality_test/utils/envs.py

```diff
@@ -0,0 +1,135 @@
+"""
+Helper functions for loading environment settings.
+"""
+
+# import json
+import os
+import sys
+# import subprocess
+from time import sleep
+from path import Path as path
+
+
+def repo_root():
+    """
+    Get the root of the git repository (edx-platform).
+
+    This sometimes fails on Docker Devstack, so it's been broken
+    down with some additional error handling.  It usually starts
+    working within 30 seconds or so; for more details, see
+    https://openedx.atlassian.net/browse/PLAT-1629 and
+    https://github.com/docker/for-mac/issues/1509
+    """
+    file_path = path(__file__)
+    attempt = 1
+    while True:
+        try:
+            absolute_path = file_path.abspath()
+            break
+        except OSError:
+            print(f'Attempt {attempt}/180 to get an absolute path failed')
+            if attempt < 180:
+                attempt += 1
+                sleep(1)
+            else:
+                print('Unable to determine the absolute path of the edx-platform repo, aborting')
+                raise
+    return absolute_path.parent.parent.parent.parent
+
+
+class Env:
```

Comment:
I suspect that many of these env vars are unused. Could you grep for each one in the js_test.py file and drop the ones that are not referenced?